src/runtime/recipe/recipe-walker.ts

```diff
@@ -23,15 +23,15 @@ export class RecipeWalker extends Walker {
   // tslint:disable-next-line: no-any
   onHandle?(recipe: Recipe, handle: Handle): any;
   // tslint:disable-next-line: no-any
```

Comment:
Are these void rather than any?

Reply:
No,  I think they can return like a list of functions, _or_ a function, _or_ void.